src/modalities/training/gradient_clipping/fsdp_gradient_clipper_config.py

```diff
@@ -26,7 +26,27 @@ class FSDPGradientClipperConfig(BaseModel):
     wrapped_model: PydanticPytorchModuleType
 
 
-class FSDPDummyGradientClipperConfig(BaseModel):
+class FSDP2GradientClipperConfig(FSDP1GradientClipperConfig):
```

Comment:
This is an "artificial" dependency in my opinion. I would just add
```
    max_norm: Annotated[float, Field(strict=True, gt=0)]
    norm_type: GradientClippingMode
    wrapped_model: PydanticPytorchModuleType
```
and remove the dependency.